test/testdata/disabled/whitequark/parse_encoding_.rb

```diff
@@ -1,3 +0,0 @@
-# typed: true
-
-__ENCODING__
```

Comment:
instead of deleting this test, can you move it to `test/testdata/whitequark/parse_encoding.rb`?